### PR TITLE
new Travis job to show current issues with Symfony 3.3-dev

### PR DESCRIPTION
**Pull request by craue**

### CODE REVIEW

**Comment by mikeSimonson** [comment]
@craue Can you put it in allowed failure until it's released ?

Thanks

**Comment by craue** [comment]
Just wanted to make you aware of these issues before it's actually released. But sure, will do.

File: .travis.yml [hunk 1]
```diff
@@ -20,8 +20,11 @@ matrix:
       env: deps=low SYMFONY_DEPRECATIONS_HELPER=weak
     - php: 5.6
       env: SYMFONY_VERSION="2.8.*@dev"
+    - php: 5.6
+      env: SYMFONY_VERSION="3.3.*@dev"
   allow_failures:
     - php: nightly
+    - env: SYMFONY_VERSION="3.3.*@dev"
```
Comment (stof) [inline .travis.yml]: don't enforce ``3.3.*@dev``, which will force us to maintain the list every time.

Instead, add a job changing the min stability to test against dev version of all vendors. See https://github.com/minkphp/Mink/blob/9ea1cebe3dc529ba3861d87c818f045362c40484/.travis.yml to see how to do it

Comment (craue) [inline .travis.yml]: done

**Comment by mikeSimonson** [comment]
@craue Thanks

**Pull request merged**